apps/address-book/app/components/Panel/NewEntity.js

```diff
@@ -4,11 +4,13 @@ import { Form, FormField } from '../Form'
 import { DropDown, TextInput } from '@aragon/ui'
 import web3Utils from 'web3-utils'
 
-const ENTITY_TYPES = [ 'Individual', 'Organization', 'Project' ]
+const ENTITY_TYPES = [ 'Individual', 'Organization', 'Custom type…' ]
 const INITIAL_STATE = {
   name: '',
   address: '',
   type: 'Individual',
+  isCustomType: false,
```

Comment:
If types are coming from appState, we don't need special handling here for `isCustomType`